docs/self-managed/connectors-deployment/install-and-start.md

```diff
@@ -0,0 +1,31 @@
+---
+id: install-and-start
+title: Installation
+description: "Let's get started with Connectors by installing and running them."
+---
+
+The concept of a [Connector](/components/integration-framework/introduction.md#connectors) consists of two parts:
+
+- The business logic is implemented by a [Connector function](/components/integration-framework/connectors/custom-built-connectors/connector-sdk.md#runtime-logic)
+  and executed by a [Connector runtime environment](/components/integration-framework/connectors/custom-built-connectors/connector-sdk.md#runtime-environments).
+- The user interface during modeling is provided using a [Connector template](/components/integration-framework/introduction.md#connectors).
+
+Please refer to the [Installation Guide](/self-managed/platform-deployment/overview.md) for details on how to install Connector business logic.
```

Comment:
Well, to be consistent, we point to the different options for installing the backend which is usually either Kubernetes, Docker, or Manual installation. Currently we support Docker and Manual. There is a note on the Kubernetes section that says "Web Modeler not yet included". We could maybe add the Connectors there. It is consistent for all components then.

Reply:
Yeah, then let's do that.

My rationale is: lets be explicit about what user can and cannot do. Otherwise they might be frustrated (_What? Only docker-compose is possible?_)

Reply:
I agree 👍

Reply:
Fixed with [49833b0](https://github.com/camunda/camunda-platform-docs/pull/1376/commits/49833b01270769edc560abe2c9ff202f16372efb)